pkg/testgen/testgen.go

```diff
@@ -0,0 +1,163 @@
+package main
+
+import (
+	"fmt"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+    "text/template"
+    "os"
+    "os/exec"
+    path "path/filepath"
+    "strings"
+
+    //"github.com/tonnerre/golang-pretty"
+
+)
+
+// generated test files use this prefix
+const prefix string = "gen_"
+
+var builtinTypes = map[string]struct{}{
+    "uint": {}, "uint8": {}, "uint16": {}, "uint32": {}, "uint64": {},
+    "int": {}, "int8": {}, "int16": {}, "int32": {}, "int64": {},
+    "float32": {}, "float64": {},
+    "complex64": {}, "complex128": {},
+    "string": {}, "byte": {}, "rune": {},
+    "chan": {}, "bool": {},
+    "uintptr": {}, "error": {},
+}
+
+func isBuiltin(s string) bool {
```

Comment:
Not needed once you replace the `{{BuiltinOrIdent .Expected}}` with `reflect.DeepEqual``. The same for the `builtinTypes`.